static/src/stylesheets/module/email/_form.scss

```diff
@@ -7,6 +7,11 @@
     // adjustments, off grid
     width: calc(100% + 8px);
     margin-left: -4px;
+
+    @include mq($from: mobileLandscape, $until: tablet) {
+        margin-right: 50%;
+        width: calc(50% + 8px);
```

Comment:
we have a few instances of calc in the codebase, mainly commercial but not exclusively. given we trigger layout on scroll relentlessly, i wonder how much difference it makes? would be v interesting to see what diff it makes to scroll without them on an older device...